sub-folder-06/index.md

```diff
@@ -0,0 +1,21 @@
+# This is for homework
+
+## It is the index file
+
+1. This is the first item in the list
+2. This is the second item
+3. and the third
+4. and finally the fourth
+
+class Cat:
+    
+    def __init__(self, name):
+        self.name = name
```

Comment:
there is a specific way to mark up code so that the whole code block is properly displayed.